frontend/server/src/DAO/ProblemsetIdentityRequest.php

```diff
@@ -34,14 +37,17 @@ public static function getFirstAdminForProblemsetRequest(
             WHERE
                 r.problemset_id = ?;';
 
-        /** @var list<array{accepted: bool|null, admin_id: int|null, extra_note: null|string, identity_id: int, last_update: null|string, problemset_id: int, request_time: string}> */
+        /** @var list<array{accepted: bool|null, admin_id: int|null, extra_note: null|string, identity_id: int, last_update: null|string, problemset_id: int, request_time: string}>|null */
```

Comment:
```suggestion
        /** @var list<array{accepted: bool|null, admin_id: int|null, extra_note: null|string, identity_id: int, last_update: null|string, problemset_id: int, request_time: string}> */
```

`GetAll()` no regresa nulo bajo ninguna circunstancia.

Reply:
Listo. Se quita el `null`